faunadb-scala/src/main/scala/faunadb/values/Codec.scala

```diff
@@ -35,6 +40,30 @@ object Decoder {
       }
   }
 
+  implicit object CharDecoder extends Decoder[Char] {
+    def decode(v: Value, path: FieldPath) =
+      v match {
+        case LongV(l) => Result.successful(l.toChar, path)
+        case v => Result.Unexpected(v, "Long", path)
```

Comment:
What's the final error message? Isn't it confusing that we're trying to decode a char and the error message mentions a `Long`? Idk...